README.md

```diff
@@ -36,30 +37,7 @@ _Draft support has only been added for these actions: Scaling commands, Update C
     ![deployDraft](resources/readme/deployDraft.png)
 1. Once your deployment has completed your changes should accurately be reflected in the Azure view!
 
-<!-- region exclude-from-marketplace -->
```

Comment:
Is there a reason you removed this sections from the changelog?

Reply:
Yeah, we need to keep this.

Reply:
Must've done this on accident